client_linux_test.go

```diff
@@ -798,6 +798,81 @@ func TestDestinations_Unpack(t *testing.T) {
 	}
 }
 
+func TestConfig(t *testing.T) {
+	fn := func(gerq genetlink.Message, _ netlink.Message) ([]genetlink.Message, error) {
+		msg := []genetlink.Message{
+			{
+				Header: genetlink.Header{
+					Command: cipvs.CmdGetConfig,
+					Version: 1,
+				},
```

Comment:
We don't return `Header` in other test cases, so I think it can be remove here. (It's also wrong as the ip_vs module replies with command set to `cipvs.CmdSetConfig`)

Reply:
Ah you are right, wrongly added this here 🤦